### ISSUE TITLE
Report file empty

### ISSUE THREAD

**Issue by rzeAkbari** [go-gremlins/gremlins#186]
### Bug Description

when running gremlins unleashed command on main.go file, the report file is empty.

### To Reproduce

1. Execute Gremlins unleash on main.go in the cmd /gremlins: gremlins unleash -o test.json2. 
I see the correct log in the terminal
```
Starting...
Gathering coverage... done in 330.058424ms
 NOT COVERED CONDITIONALS_NEGATION at main.go:46:9

Mutation testing completed in 82 milliseconds 751 microseconds
Killed: 0, Lived: 0, Not covered: 1
Timed out: 0, Not viable: 0
Test efficacy: NaN%
Mutator coverage: 0.00%
ERROR: below mutant coverage-threshold

```
### Found behaviour
test.json file is empty

### Expected behaviour
test.json should contain the a json with the correct info

### Operating System

- OS: Mac

**Comment by k3rn31** [comment]
Hello @rzeAkbari. 
This has to be expected because main.go has no coverage. Maybe we can improve the reports when there is no actual coverage.

**Comment by k3rn31** [comment]
@rzeAkbari can you take care of this?

**Comment by rzeAkbari** [comment]
gladly :)

**Comment by rzeAkbari** [comment]
The problem comes form a division by zero issue. In case we have only not covered mutations report, efficacy becomes NAN, hence facing marshalling issues later on.
```
rep.tEfficacy = float64(rep.killed) / float64(rep.killed+rep.lived) * 100
```
tests efficacy should be calculated only if the number of killed is grater than zero, otherwise it keeps its zero value, so we avoid division by zero. 
does this make sense to you @k3rn31 ?

**Comment by k3rn31** [comment]
Closed by #188
